finch.yaml

```diff
@@ -90,8 +90,6 @@ mounts:
 # `/mnt/lima-${VOLUME}`.
 # 🟢 Builtin default: null
 # For Finch, this value should always be the same as the diskName in pkg/disk/disk.go
-additionalDisks:
-- "finch"
```

Comment:
Is this not required anymore?

Reply:
+1, maybe a comment here directing readers to additionalDisks being set in lima_config_applier?

Reply:
Yeah, this is now handled in `lima_config_applier`. The idea is to move any platform/config specific options to the `config_applier`s, so that we only have one "base" YAML file.

The problem with a comment is that it will be persisted to the output finch.yaml, and that might be confusing. I can add it if you really think it'll help though.